views/options/settings.jade

```diff
@@ -88,10 +88,30 @@ script(type='text/ng-template', id='partials/options.settings.settings.html')
           .panel-heading
             span Registration
           .panel-body
-            p(ng-if='user.auth.facebook.id') Registered with Facebook
+            p(ng-if='user.auth.facebook.id') 
+              =env.t('registeredWithFb')
             div(ng-if='user.auth.local.username')
-              p Username: {{user.auth.local.username}}
-              p Email: {{user.auth.local.email}}
+              p
+              =env.t('username') 
+              |: {{user.auth.local.username}}
+              p
+              small
```

Comment:
I do. It looks really cluttered, and .muted might help with that.

On Mon, Jan 12, 2015 at 8:23 PM, Blade Barringer <notifications@github.com>
wrote:

> In views/options/settings.jade
> <https://github.com/HabitRPG/habitrpg/pull/4504#discussion-diff-22842812>:
>
> >              div(ng-if='user.auth.local.username')
> > -              p Username: {{user.auth.local.username}}
> > -              p Email: {{user.auth.local.email}}
> > +              p
> > +              =env.t('username')
> > +              |: {{user.auth.local.username}}
> > +              p
> > +              small
>
> Anyone else think all these small elements should have a .muted class
> applied to them?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/HabitRPG/habitrpg/pull/4504/files#r22842812>.
>